coalib/__init__.py

```diff
@@ -1,3 +1,11 @@
+"""
+The coalib package is a collection of various subpackages
+regarding writing and editing bears.
+Various other packages such as formatting and
+settings are also included in coalib.
```

Comment:
your way of breaking the line is pretty weird here, can't you break after 80 chars consistency? Just set it in your editor.